src/utils/get-form.js

```diff
@@ -47,7 +47,7 @@ export function getFormStateKey(state, model, s = defaultStrategy, currentPath =
       return false;
     }
 
-    if (s.isObject(subState)) {
+    if (state !== subState && s.isObject(subState)) {
```

Comment:
The other possibility is that at line 20 we just check if key === "" and return false, but that potentially changes the getFormStateKey logic (even though all test cases still pass).  I would mildly prefer that over this safer change since other functions like joinPaths doesn't work with empty path segments.

Reply:
I'd prefer that too.